module/data/activity/base-activity.mjs

```diff
@@ -686,7 +686,7 @@ export default class BaseActivityData extends foundry.abstract.DataModel {
       let actionType = this.actionType;
       if ( (actionType === "mwak") && rollConfig.attackMode?.startsWith("thrown") ) actionType = "rwak";
       const bonus = foundry.utils.getProperty(this.actor ?? {}, `system.bonuses.${actionType}.damage`);
-      if ( bonus && (parseInt(bonus) !== 0) ) parts.push(bonus);
+      if ( bonus && (Number(bonus) !== 0) ) parts.push(bonus);
```

Comment:
Number behaves differently than `parseInt`, so `Number("2 + 2")` results in `NaN`, not `4`.

The intention here is to detect `"0"` or a blank string, so we could just do the `bonus !== "0"`.

Reply:
Well NaN is fine, then. The important thing is that it is not `"0"` (or `"000"` repeating).